packages/sit-onyx/src/components/OnyxButton/OnyxButton.ct.tsx

```diff
@@ -0,0 +1,17 @@
+import { expect, test } from "../../playwright-axe";
+import OnyxButton from "./OnyxButton.vue";
+
+test("should render", async ({ mount, makeAxeBuilder }) => {
+  // ARRANGE
+  const component = await mount(<OnyxButton label="Button" color="secondary" />);
```

Comment:
```suggestion
  const component = await mount(<OnyxButton label="Button" />);
```

For the default button screenshot we should also use the default color :)
You can also change it in a future PR